app/controllers/kaui/subscriptions_controller.rb

```diff
@@ -228,11 +228,11 @@ def lookup_bundle_and_plan_details(subscription, base_product_name = nil)
     end
 
     def catalog_plans(product_category = nil)
-      return Kaui::Catalog.available_base_plans(options_for_klient) if product_category == 'BASE'
+      return Kaui::Catalog.available_base_plans(@subscription.account_id, options_for_klient) if product_category == 'BASE'
```

Comment:
Should we pass in the `account_id` instead?

Even in `lookup_bundle_and_plan_details`, we don't always rely on `@subscription`, except for `Kaui::Bundle.find_by_id` (which we should change for consistency).

Reply:
So you mean we need to pass account_id from lookup_bundle_and_plan_details to Kaui::Catalog.available_base_plans, right?
Instead of getting it from subscription.account_id?

Reply:
Yes - I'm looking at the other methods and we don't really use the instance variable elsewhere.